doc/examples/animations-pulsating-sphere.ipynb

```diff
@@ -0,0 +1,363 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Animations of a Pulsating Sphere"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sfs\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "from IPython.display import HTML"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this example, the sound field of a pulsating sphere is visualized.\n",
+    "Different acoustic variables, such as sound pressure,\n",
+    "particle velocity, and particle displacement, are simulated.\n",
+    "The first two quantities are computed with\n",
+    "\n",
+    "- `sfs.mono.source.pulsating_sphere()` and\n",
```

Comment:
If you want to be fancy, you can use links like this:

    [sfs.mono.source.pulsating_sphere()](../sfs.mono.source.rst#sfs.mono.source.pulsating_sphere)

This isn't really meaningful in JupyterLab, but it links to the documentation in Sphinx's HTML output.